### PR TITLE
Checkbox component

### PR DESCRIPTION
**Pull request by OxygenO22**

### CODE REVIEW

**Review by Kurilo-Pavel — APPROVED** [review]

**Review by Evgeniy121212 — APPROVED** [review]
Finally! :+1:

**Review by valer-sky — APPROVED** [review]

File: src/components/Checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, {useState} from 'react';
+import classes from './Checkbox.module.scss';
+import '../index.scss';
+
+const Checkbox = (props) => {
```
Comment (Evgeniy121212) [inline src/components/Checkbox.tsx:5]: props must have type

**Review by Evgeniy121212 — CHANGES_REQUESTED** [review]

File: src/components/Checkbox.tsx (hunk 1 above)
Comment (OxygenO22) [reply src/components/Checkbox.tsx]: done

**Review by Evgeniy121212 — APPROVED** [review]

File: src/components/Checkbox.module.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (Terapool) [inline src/components/Checkbox.module.scss:1]: remove the leading empty line

File: src/components/Checkbox.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React, {useState} from 'react';
+import classes from './Checkbox.module.scss';
+import '../index.scss';
+
+const Checkbox: React.FC = (props) => {
+    const [isChecked, setIsChecked] = useState(false);
+    return (
```
Comment (Terapool) [inline src/components/Checkbox.tsx:7]: Please, add an empty line before return

**Pull request merged**